examples/dumpmodules.rs

```diff
@@ -27,7 +28,7 @@ enum Verbose {
 
 fn print_minidump_modules<T: AsRef<Path>>(path: T,
                                           verbose: Verbose) {
-    match Minidump::read_path(path.as_ref()) {
+    match Minidump::<File>::read_path(path.as_ref()) {
```

Comment:
i have no idea why rustc needs to be told explicitly what `T` is here, despite `read_path` explicitly stating what `T` is in the return type

Reply:
https://github.com/rust-lang/rust/issues/46988

Reply:
nevermind, figured this out